pyispyb/app/extensions/auth/bearer.py

```diff
@@ -20,9 +23,18 @@ def verify_jwt(token: str):
 
 
 async def JWTBearer(
+    request: Request,
+    onetime: str = Depends(onetime),
     credentials: HTTPAuthorizationCredentials = Depends(security),
 ):
-    if credentials:
+    # One time token authentication
+    expire_onetime_tokens()
+    if onetime:
+        person_dict = validate_onetime_token(onetime, request.url.components.path)
+        set_token_data(person_dict)
+
+    # JWT authentication
+    elif credentials:
```

Comment:
Small detail, but I think this should be first. If this is missused and a request somehow ends up with both onetime token and JWT auth, the system should only look at the normal auth to avoid any confusion.

For instance, I can see this happening with a copied/pasted URL with the person not noticing there was a onetime token in the parameters.